examples/dataflow-data-generator/data-generator-pipeline/data_generator_pipeline.py

```diff
@@ -125,6 +125,19 @@ def run(argv=None):
                 )
         )
 
+    if data_args.write_to_parquet:
+        with open(data_args.schema_file, 'r') as infile:
+            str_schema = json.load(infile)
+        pa_schema = get_pyarrow_translated_schema(str_schema)
+        (rows
```

Comment:
Have you done a test writing TIMESTAMP type fields? I'm curious if we need a date handler like [`fix_record_for_avro`](https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/dataflow-data-generator/data-generator-pipeline/data_generator/AvroUtil.py#L17) to cast from a date string to a unix timestamp. The data generator is going to create this field in "yyyy/MM/ddTHH:mm:SS" see [`sanity_check`](https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/dataflow-data-generator/data-generator-pipeline/data_generator/PrettyDataGenerator.py#L331)
https://arrow.apache.org/docs/python/generated/pyarrow.timestamp.html

Reply:
Alternatively, you could change how we generate dates upstream.

Reply:
I can definitely look into this